addons/disposable/functions/fnc_takeLoadedATWeapon.sqf

```diff
@@ -16,19 +16,29 @@
 #include "script_component.hpp"
 
 params ["_unit"];
-TRACE_1("params",_unit);
+TRACE_1("params", _unit);
```

Comment:
Spaces in macros will cause them to be logged as well, adding additional spaces in the logs (you'd get double spaces).